src/bigint.rs

```diff
@@ -0,0 +1,65 @@
+#[cfg(feature = "num-bigint")]
```

Comment:
Make a submodule under `value/bigint.rs`.

Also: only needs one `#[cfg(..)]` on the `mod bigint` declaration.

Reply:
To clarify: do you mean that the directory structure should be changed to this:

```
src/value.rs -> src/value/mod.rs
src/bigint.rs -> src/value/bigint.rs
```
?

Reply:
:+1:

Reply:
It is moved now